crates/solidity/inputs/language/src/definition.rs

```diff
@@ -4588,31 +4463,6 @@ codegen_language_macros::compile!(Language(
                     title = "Yul Keywords",
                     lexical_context = Yul,
                     items = [
-                        Keyword(
-                            name = YulAbstractKeyword,
```

Comment:
Not sure I understand why was `YulAbstractKeyword` removed here?

Reply:
If unintended, I suggest doing a quick `println!()`/diff to make sure that the added and removed entries here are the same.

Reply:
Hmm... this was definitely unintended. I'll restore it and check that there are no more similar cases.

Reply:
I couldn't find any other cases of unintended removals. All other removed Yul keywords are added as built-in functions.